var/lib/delphix-platform/ansible/apply

```diff
@@ -36,13 +36,12 @@ function find_playbooks() {
 	# old directory. Once all consumers can be updated to install
 	# into the new directory, we can remove this logic.
 	#
-	if ! find "/etc/delphix-platform/ansible" \
+	find "/etc/delphix-platform/ansible" \
```

Comment:
why don't we change the `find` call a few lines up?

Reply:
Since we know that delphix-platform ships a playbook, the first invocation `find` should always find something. It seems worth asserting that this is the case.

Once we move the playbook provided by virtualization, though,  `/etc/delphix-platform/ansible` won't exist, so the second invocation of `find` will fail. It will also fail when we do the appliance-build because virtualization hasn't been installed yet when we run this script. That should be fine though, so we shouldn't check for errors in the second call.